test/performance/range/gap_decorator_rand_read_benchmark.cpp

```diff
@@ -71,10 +71,14 @@ void read_random(benchmark::State & state)
 }
 
 // Read at random position in UNGAPPED sequence
-BENCHMARK_TEMPLATE(read_random, gap_decorator<const std::vector<dna4> &>, false)->Apply(custom_arguments);
-BENCHMARK_TEMPLATE(read_random, std::vector<gapped<dna4>>, false)->Apply(custom_arguments);
+BENCHMARK_TEMPLATE(read_random, seqan3::gap_decorator<const std::vector<seqan3::dna4> &>, false)
```

Comment:
Ask the others for this, because I'm not very strong with the codestyle rules, but I think in this case it should be:

```suggestion
BENCHMARK_TEMPLATE(read_random,
                   seqan3::gap_decorator<const std::vector<seqan3::dna4> &>,
                   false)->Apply(custom_arguments);
```

But before you change anything ask someone else :)

Reply:
I would propose to make global typedef for the two types. This would make the code more readable.

```

using seqan3_gap_decorator = gap_decorator<const std::vector<dna4> &>;
using vector_gap_sequence = gap_decorator<const std::vector<dna4> &>;

// Insert gaps of length 1 from left to right into UNGAPPED sequence
BENCHMARK_TEMPLATE(insert_left2right, vector_gap_sequence, false)->Apply(custom_arguments);
BENCHMARK_TEMPLATE(insert_left2right, vector_gap_sequence, false)->Apply(custom_arguments);

// Insert gaps of length 1 from left to right into GAPPED sequence
BENCHMARK_TEMPLATE(insert_left2right, vector_gap_sequence, true)->Apply(custom_arguments);
BENCHMARK_TEMPLATE(insert_left2right, vector_gap_sequence, true)->Apply(custom_arguments);
```

@marehr what do you think?

Reply:
@marehr polite ping

Reply:
sorry for the late response, I slightly prefer to call it explicitly.

In the end both `gap_decorator<const std::vector<dna4> &>` and `std::vector<gapped<dna4>>` are gap sequences. Thus, `gap_sequence_vector` and `gap_sequence_gap_decorator` might be better names.